devel/spdlog/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake  1.1
 
-github.setup        gabime spdlog 0.16.3 v
+github.setup        gabime spdlog 1.4.2 v
 #revision            0
```

Comment:
Don't comment out the line. Leave the `revision` in at 0.

Reply:
when doing that, also please squash your commits into one